Examples/Example-iOS/AppDelegate.swift

```diff
@@ -55,7 +53,7 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
             }
         }
         let attributes = ["My Attribute":"My Attribute Value"]
-        BacktraceClient.shared?.addBreadcrumb("My Native Breadcrumb",
+        _ = BacktraceClient.shared?.addBreadcrumb("My Native Breadcrumb",
```

Comment:
To avoid warning "Result of call to 'addBreadcrumb(_:attributes:type:level:)' is unused" we add this code. Significance of _ is to remove unused warning.